frontend/src/tests/session-tests.js

```diff
@@ -76,12 +78,12 @@ export function sessionsTests(api) {
         expect(withNewSessions.map(x => x.id)).toContain(createdSession.id);
 
         // save this session for use in later tests
```

Comment:
Probably don't need this command anymore?

Reply:
sure